apps/prairielearn/src/pages/partials/editQuestionPointsForm.ejs

```diff
@@ -6,14 +6,14 @@
     <button type="button" class="btn btn-secondary" onclick="$('#<%= id %>').popover('hide')">Cancel</button>
   </div>
 <% } else { %>%>
-<form name="edit-question-points-form" method="POST">
+<form name="edit-question-points-form" method="POST" class="needs-validation">
```

Comment:
I'm trying to figure out what the `needs-validation` class does. AFAICT from the Bootstrap docs, this is half of an example showing how to use JavaScript to perform custom validation, and I can't find any such JS in PrairieLearn.

Reply:
Indeed it was not needed. Removed in abb9cd.